src/AutoRest.CSharp.V3/Pipeline/Extensions.cs

```diff
@@ -58,15 +66,29 @@ internal static class Extensions
             _ => null
         };
 
-        private static readonly Func<string, bool, string?> NumberSerializer =
-            (vn, nu) => $"writer.WriteNumberValue({vn}{(nu ? ".Value" : string.Empty)});";
-        private static Func<string, bool, string?> StringSerializer(bool includeToString = false) =>
-            (vn, nu) => $"writer.WriteStringValue({vn}{(includeToString ? ".ToString()" : string.Empty)});";
+        private static readonly Func<string, bool, FrameworkTypeReferenceFormat, string?> NumberSerializer =
+            (vn, nu, f) => $"writer.WriteNumberValue({vn}{(nu ? ".Value" : string.Empty)});";
+        private static Func<string, bool, FrameworkTypeReferenceFormat, string?> StringSerializer(bool includeToString = false) =>
+            (vn, nu, f) => $"writer.WriteStringValue({vn}{(includeToString ? ".ToString()" : string.Empty)});";
+
+        private static string? ToFormatSpecifier(this FrameworkTypeReferenceFormat format) => format switch
+        {
+            FrameworkTypeReferenceFormat.DateTimeRFC1123 => "R",
+            FrameworkTypeReferenceFormat.DateTimeISO8601 => "yyyy-MM-ddTHH:mm:ssZ",
+            FrameworkTypeReferenceFormat.Date => "yyyy-MM-dd",
```

Comment:
These formats are the format of the datetimes of the testserver tests. Do we even know if these are the right formats for all swaggers?

Reply:
Nope, we don't. We'll eventually figure out if we need more.